src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/Documents/TextRangeEditTables.cs

```diff
@@ -1180,7 +1180,7 @@ internal static TextRange InsertRows(TextRange textRange, int rowCount)
                     {
                         // Create new cell and transfer all formatting properties to it from a source cell.
                         // All properties except for RowSpan will be transferred.
-                        AddCellCopy(newRow, currentCell, -1/*negative means at the endPosition*/, /*copyRowSpan:*/false, /*copyColumnSpan:*/true);
+                        AddCellCopy(newRow, currentCell, -1/*negative means at the endPosition*/, copyRowSpan: false, copyColumnSpan: true);
```

Comment:
Maybe this was skipped on purpose, but if not: `negative means at the endPosition` -> `cellInsertionIndex`